test/test-network/systemd-networkd-tests.py

```diff
@@ -5878,6 +5878,61 @@ def test_dhcp6pd(self):
 
         self.teardown_nftset('addr6', 'network6', 'ifindex')
 
+    def test_dhcp6pd_dbus_prefix(self):
```

Comment:
Please merge the test with test_dhcp6pd.

Reply:
Done.